bin/alluxio-start.sh

```diff
@@ -131,8 +131,22 @@ do_mount() {
   MOUNT_FAILED=0
   case "$1" in
     Mount|SudoMount)
-      ${LAUNCHER} "${BIN}/alluxio-mount.sh" $1
-      MOUNT_FAILED=$?
+      local tier_alias=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.alias)
+      local tier_path=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.path)
+
+      if [[ ${TIER_ALIAS} != "MEM" ]]; then
```

Comment:
I think this needs to be lowercase `tier_alias`

Reply:
You are correct